ci/gpu/build.sh

```diff
@@ -80,7 +80,7 @@ gpuci_conda_retry install -y \
                   "rapids-notebook-env=$MINOR_VERSION.*" \
                   "dask-cuda=${MINOR_VERSION}" \
                   "rmm=$MINOR_VERSION.*" \
-                  "ucx-py=${MINOR_VERSION}"
+                  "ucx-py=0.21.0a*"
```

Comment:
@jakirkham @galipremsagar, do we need the `.0a` suffix here? or can it just be `0.21.*`?

Reply:
I think `0.21.*` should be ok

Reply:
Done 👍

Reply:
Thanks Prem 🙂